changelog/changes/2022-04-07-google-gve.md

```diff
@@ -0,0 +1 @@
+- Enabled GVE kernel driver, which adds support for Google Virtual NIC on GCP ([PR#1802](https://github.com/flatcar-linux/coreos-overlay/pull/1802))
```

Comment:
```suggestion
- GCE: Enabled GVE kernel driver, which adds support for Google Virtual NIC on GCP ([PR#1802](https://github.com/flatcar-linux/coreos-overlay/pull/1802))
```
Release notes are skimmed and I think it helps to have the OEM prefixes

Reply:
will take note for the future.